test/powershell/Modules/Microsoft.PowerShell.Core/Job.Tests.ps1

```diff
@@ -0,0 +1,76 @@
+Describe "Job Cmdlet Tests" -Tag "CI" {
```

Comment:
`pending` is a way that we can track those tests which will be eventually enabled. We will start tracking Pending tests and drive that count to 0 over time. A skipped test is a test which isn't really ever applicable (for example, a test that is only applicable to Linux, Core. or windows).